cmd/restic/integration_test.go

```diff
@@ -1410,7 +1410,7 @@ func TestPrune(t *testing.T) {
 		testPrune(t, opts, checkOpts)
 	})
 	t.Run("Small", func(t *testing.T) {
-		opts := PruneOptions{MaxUnused: "100%", RepackSmall: true}
```

Comment:
`MaxUnused: "100%"` does no longer work, you have to specify `MaxUnused: "unlimited"`

Reply:
Ahh, gotcha.  Implementing change.